doc/manual/getting_started_mgmt.rst

```diff
@@ -57,10 +57,16 @@ Return to the terminal where the master is running. You should see an output sim
 In other words, a worker created by the master has executed the experiment and carried out the print instruction. Congratulations!
 
 .. tip::
+    
+    In order to run the master and the clients on different PCs, please make sure your PC's and network's firewall do not block traffic on the ports used. Or you may try using SSH port forwarding.
```

Comment:
I think the SSH note just causes confusion, also SSH is typically not available on Windows - and Windows users are typically the most clueless. I suggest simply removing it.